Qiy Scheme Documentation.md

```diff
@@ -0,0 +1,72 @@
+
+A Work Stream prepares drafts concerning the management, further development and maintenance of the Qiy Scheme for consideration by the Supervisory Authority.
+
+A Work Stream may establish working groups, if required. When this is the case, the Work Stream shall decide on the rules for the working group, within the scope of the Qiy Foundation�s Rules & Regulations and the Work Stream Procedures.
+
+Participation in a Work Stream or working group is by invitation by the Chairperson only. Non-Members with a special expertise may be invited to participate in working groups or even in a Work Stream. Where deemed necessary by the Chairperson of a Work Stream, participants in Work Streams or working groups will be asked to sign a Mutual Confidentiality and Non-Disclosure Agreement with the Qiy Foundation.
+
+Counselors of the Qiy Foundation may attend meetings of a Work Stream and participate in the work without the right to vote.
+
+The Chairpersons of each Work Stream ensure that all pull requests suggested by Contributors are reviewed in the Work Stream.
+
+The Work Stream Procedures adopted by the Executive Board of the Qiy Foundation apply.
+
+### Review Board
+It is the task of the Review Board to review the work of the different Work Streams and to seek the advice of both the User Voice and the Council of Regional Authorities. Where necessary, the Review Board will ask the Work Stream to reconsider their proposals in the light of the advice received. The procedural rules for the adoption of Work Stream pull requests apply.
+
+Members of the Review Board are nominated by the Executive Board of the Qiy Foundation, but selected and appointed by the Supervisory Authority. The Rules on the Selection and Appointment of members of the Review Board apply.
+
+### User Voice
+The User Voice is an advisory panel of the Qiy Foundation. Its members represent the interests of the different user groups of the Qiy Scheme. Its role is to advise the Supervisory Authority on proposals made by Work Streams.
+
+The Rules on the Selection and Appointment of members of the User Voice apply.
+
+### Council of Regional Authorities
+The Council of Regional Authorities consists of representatives of the Regional Authorities. Its members represent the interests of the Regional Authorities. Its role is to advise the Supervisory Authority on proposals made by Work Streams.
+
+The Rules on the Selection and Appointment of members of the Council of Regional Authorities apply.
+
+### Supervisory Authority
+The Supervisory Authority is responsible for the management, further development and maintenance of the Qiy Scheme. Once it has accepted a new deliverable for the Qiy Scheme, it shall instruct the Review Board to merge the deliverable with the existing version of the Qiy Scheme. It will also assign a version number to each new deliverable.
```

Comment:
The  Executive Board is responsible for the management, further development and maintenance of the Qiy Scheme. Once it has accepted a new deliverable for the Qiy Scheme, it shall instruct the Review Board to merge the deliverable with the existing version of the Qiy Scheme. It will also assign a version number to each new deliverable.

 The Rules on the Selection and Appointment of members of the Executive Board apply.